.github/ISSUE_TEMPLATE/generic_issue.yaml

```diff
@@ -0,0 +1,126 @@
+name: Report a generic issue
+description: Report a generic issue related to Home Assistant Intents.
+body:
+  - type: markdown
+    attributes:
+      value: |
+        This issue form is for reporting issues related to Home Assistant Intents
+
+        If you have a feature or enhancement request, please use the [feature request][fr] section of our [Community Forum][fr].
+
+        [fr]: https://community.home-assistant.io/c/feature-requests
+  - type: dropdown
```

Comment:
We shouldn't do it like this, instead users should use labels. Because labels will trigger code owners being mentioned.

Reply:
Although, maybe we can have the bot pick this up? Not sure what's the best user experience. Obviously labels means we don't have to keep this file up to date, but UX might be worse.

Reply:
We should not support multiple languages. It makes everything more complicated. Just let users mention it in their "additional info".

Agree on the test.